cookie_consent/util.py

```diff
@@ -151,12 +151,10 @@ def get_cookie_string(cookie_dic):
     """
     Returns cookie in format suitable for use in javascript.
     """
-    expires = datetime.datetime.now() + datetime.timedelta(
-        seconds=settings.COOKIE_CONSENT_MAX_AGE)
     cookie_str = "%s=%s; expires=%s; path=/" % (
```

Comment:
```suggestion
    cookie_str = "%s=%s; max-age=%s; path=/" % (
```
this should just set max-age instead of `expires`, the former is now supported by all browsers.